python/mdvtools/server.py

```
@@ -31,6 +31,7 @@
 from mdvtools.conversions import convert_scanpy_to_mdv
 from mdvtools.llm.chat_server_extension import chat_extension
 routes = set()
+from mdvtools.socketio_upload import initialize_socketio_upload, register_project_for_upload
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)